apps/spruce/src/pages/adminSettings/index.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
+import { Link } from "react-router-dom";
+import { usePageTitle } from "@evg-ui/lib/hooks/usePageTitle";
+import {
+  SideNav,
+  SideNavGroup,
+  SideNavItem,
+  SideNavPageContent,
+  SideNavPageWrapper,
+} from "components/styles";
+import { AdminSettingsProvider } from "./Context";
+import { AdminSettingsTab } from "./Tabs";
+
+const AdminSettings: React.FC = () => {
+  usePageTitle("Admin Settings");
+
+  return (
+    <AdminSettingsProvider>
+      <SideNavPageWrapper>
+        <SideNav aria-label="Admin Settings" widthOverride={250}>
+          {}
+          <ButtonsContainer>{}</ButtonsContainer>
+
+          <SideNavGroup
+            glyph={<Icon glyph="Settings" />}
+            header="Admin Settings"
+          >
+            {}
+            <SideNavItem
+              as={Link}
+              data-cy="navitem-admin-general"
+              to="/admin-settings"
+            >
+              Announcements
+            </SideNavItem>
+          </SideNavGroup>
+
+          <SideNavGroup glyph={null} header="Restart Tasks">
+            {}
+          </SideNavGroup>
+          <SideNavGroup glyph={null} header="Tasks Logs">
+            {}
+          </SideNavGroup>
+        </SideNav>
+
+        <SideNavPageContent data-cy="admin-settings-page">
+          <AdminSettingsTab />
```

Comment:
Nit: pluralize for consistency
```suggestion
          <AdminSettingsTabs />
```